src/__internal__/sticky-footer/sticky-footer.component.tsx

```diff
@@ -23,28 +23,25 @@ const StickyFooter = ({
   const [isSticky, setIsSticky] = useState(true);
   const footerRef = useRef<HTMLDivElement>(null);
 
-  const checkFooterPosition = useCallback(
-    throttle(() => {
-      const content = containerRef.current;
-      /* Fallback to 0 to satisfy the TypeScript compiler */
-      /* footerRef will never be null in this method */
-      /* istanbul ignore next */
-      const stickyOffset = footerRef.current
-        ? footerRef.current.clientHeight / 2
-        : 0;
-      let fullyScrolled;
+  const checkFooterPosition = throttle(() => {
+    const content = containerRef.current;
```

Comment:
**question:** May I double check the reason behind us no longer making this function a React callback? 

As this function is a dependency of the effect on line 46, this effect will be called every render as `checkFoterPosition` will be redefined each time

Reply:
Valid question. I have just taken another look at this. Can't remember exactly what the lint warning I was getting was at the time but I have just revisited this. I am able to add in the `useCallback` with a dependancy array and the warning no longer seems to be there. I'll get it added back in 😄